### PR TITLE
mysql_user: add session_vars argument

### LINKED ISSUE #478: Support the session variable wsrep_on = off for the Galera cluster
##### SUMMARY

The session variable `wsrep_on = off` is useful to update db users without affecting other Galera cluster nodes.

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME

mysql.user

##### ADDITIONAL INFORMATION

https://galeracluster.com/library/documentation/mysql-wsrep-options.html#wsrep-on

```yaml

```

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Fixes https://github.com/ansible-collections/community.mysql/issues/478

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

**Comment by Andersson007** [comment]
@janggwan-im i created this PR to add the feature you need.
Do you have time to test this? Here's the short [guide](https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_test_pr_locally.html).
I think it's safe to merge it w/o manual testing as i added a couple of integration tests but i'd like to be sure this will solve your issue

File: tests/integration/targets/test_mysql_user/tasks/main.yml [hunk 1]
```diff
@@ -62,13 +62,33 @@
...
+    - name: create mysql user trying to set global variable which is forbidden
+      mysql_user:
+        <<: *mysql_params
+        name: '{{user_name_1}}'
+        password: '{{user_password_1}}'
+        state: present
+        session_vars:
+          max_connections: 1000
+      register: result
+      ignore_errors: yes
```
Comment (russoz) [inline tests/integration/targets/test_mysql_user/tasks/main.yml]:
```suggestion
      ignore_errors: true 
```

**Review by russoz — APPROVED** [review]
Tiny nit pick, other than that LGTM

File: tests/integration/targets/test_mysql_user/tasks/main.yml (hunk 1 above)
Comment (Andersson007) [reply tests/integration/targets/test_mysql_user/tasks/main.yml]: yes, good point, thanks, committed:)

**Comment by Andersson007** [comment]
@russoz thanks for reviewing!

**Pull request merged**